### PR TITLE
Add reference into the bibliography.

### PR DESCRIPTION
**Pull request by martha-thomae**

### CODE REVIEW

File: resources/mei_bibliography.bib [hunk 1]
```diff
@@ -128,6 +128,15 @@ @article{Crawford_2016
...
+ abstract = {This paper describes the challenges involved in designing a computer encoding system for early plainchant notation (both with and without staff lines) and, in particular, Old Hispanic neume notation. We have been working on a new version of the Music Encoding Initiative (MEI) schema for neume notations since 2012. Our goal and that of MEI is to digitally represent as accurately as possible the notation in musical sources, especially its semantics wherever possible, so that the encoded files can be used for various musical research purposes, such as creating critical editions and data mining. Of the early plainchant notations, Old Hispanic neume notation is the least understood. We will present some of our solutions to resolve the specific issues surrounding Old Hispanic notation while maintaining compatibility with the encoding of other styles of plainchant notation.},
+ author = {De Luca, Elsa and Bain, Jennifer and Behrendt, Inga and Fujinaga, Ichiro and Helsen, Katherine and Ignesti, Alessandra and Lacoste, Debra and Long, Sarah},
+ year = {2019},
+ title = {Capturing Early Notations in MEI: The Case of Old Hispanic Neumes},
+ pages = {229--49},
+ volume = {3},
```
Comment (musicEnfanthen) [inline resources/mei_bibliography.bib:137]:
```suggestion
 volume = {34},
 number = {3},
```

File: resources/mei_bibliography.bib [hunk 2]
```diff
@@ -128,6 +128,15 @@ @article{Crawford_2016
...
+ journal = {Musiktheorie-Zeitschrift für Musikwissenschaft}
```
Comment (musicEnfanthen) [inline resources/mei_bibliography.bib:138]:
```suggestion
 journal = {Musiktheorie. Zeitschrift für Musikwissenschaft}
```

File: resources/mei_bibliography.bib [hunk 3]
```diff
@@ -128,6 +128,15 @@ @article{Crawford_2016
...
+ abstract = {This paper describes the challenges involved in designing a computer encoding system for early plainchant notation (both with and without staff lines) and, in particular, Old Hispanic neume notation. We have been working on a new version of the Music Encoding Initiative (MEI) schema for neume notations since 2012. Our goal and that of MEI is to digitally represent as accurately as possible the notation in musical sources, especially its semantics wherever possible, so that the encoded files can be used for various musical research purposes, such as creating critical editions and data mining. Of the early plainchant notations, Old Hispanic neume notation is the least understood. We will present some of our solutions to resolve the specific issues surrounding Old Hispanic notation while maintaining compatibility with the encoding of other styles of plainchant notation.},
+ author = {De Luca, Elsa and Bain, Jennifer and Behrendt, Inga and Fujinaga, Ichiro and Helsen, Katherine and Ignesti, Alessandra and Lacoste, Debra and Long, Sarah},
+ year = {2019},
+ title = {Capturing Early Notations in MEI: The Case of Old Hispanic Neumes},
+ pages = {229--49},
```
Comment (musicEnfanthen) [inline resources/mei_bibliography.bib:136]:
```suggestion
 pages = {229--249},
```

**Comment by martha-thomae** [comment]
Thank you, @musicEnfanthen!

**Comment by musicEnfanthen** [comment]
Thank you!

**Pull request merged**